test/tab-size.js

```diff
@@ -0,0 +1,373 @@
+        });
+
+        it('should return 5 for tab-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(lessStringTabIndented, options);
+            assert.equal(startCol, 5);
+        });
+    });
+
+    describe('{syntax:\'sass\'}', function() {
+        var options = {syntax: 'sass'};
+
+        it('should return 2 for tab-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(sassStringTabIndented, options);
+            assert.equal(startCol, 2);
+        });
+    });
+
+    describe('{syntax:\'sass\', tabSize:0}', function() {
+        var options = {syntax: 'sass', tabSize: 0};
+
+        it('should return 5 for tab-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(sassStringTabIndented, options);
+            assert.equal(startCol, 2);
+        });
+    });
+
+    describe('{syntax:\'sass\', tabSize:1}', function() {
+        var options = {syntax: 'sass', tabSize: 1};
+
+        it('should return 2 for tab-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(sassStringTabIndented, options);
+            assert.equal(startCol, 2);
+        });
+    });
+
+    describe('{syntax:\'sass\', tabSize:2}', function() {
+        var options = {syntax: 'sass', tabSize: 2};
+
+
+        it('should return 3 for tab-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(sassStringTabIndented, options);
+            assert.equal(startCol, 3);
+        });
+    });
+
+    describe('{syntax:\'sass\', tabSize:4}', function() {
+        var options = {syntax: 'sass', tabSize: 4};
+
+        it('should return 5 for tab-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(sassStringTabIndented, options);
+            assert.equal(startCol, 5);
+        });
+    });
+
+    describe('{syntax:\'scss\'}', function() {
+        var options = {syntax: 'scss'};
+
+        it('should return 15 for non-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(scssStringNonIndented, options);
+            assert.equal(startCol, 15);
+        });
+
+        it('should return 2 for tab-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(scssStringTabIndented, options);
+            assert.equal(startCol, 2);
+        });
+    });
+
+    describe('{syntax:\'scss\', tabSize:0}', function() {
+        var options = {syntax: 'scss', tabSize: 0};
+
+        it('should return 15 for non-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(scssStringNonIndented, options);
+            assert.equal(startCol, 15);
+        });
+
+        it('should return 5 for tab-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(scssStringTabIndented, options);
+            assert.equal(startCol, 2);
```

Comment:
I have already explained the functionality of the tests in my previous comments.